embarkConfig/contracts.js

```diff
@@ -152,6 +152,37 @@ module.exports = {
   // used with "embark run testnet"
   testnet: {},
 
+  // merges with the settings in default
+  // used with "embark run ropsten"
+  ropsten: {
```

Comment:
Couldn't we share most of the configs with Rinkeby and only change the host? The tracking file can be the same too